homeassistant/components/ecobee/sensor.py

```diff
@@ -10,27 +12,71 @@
     SensorStateClass,
 )
 from homeassistant.config_entries import ConfigEntry
-from homeassistant.const import PERCENTAGE, TEMP_FAHRENHEIT
+from homeassistant.const import (
+    PERCENTAGE,
+    TEMP_FAHRENHEIT,
+    CONCENTRATION_PARTS_PER_MILLION,
+    CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+)
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers.entity import DeviceInfo
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 
 from .const import DOMAIN, ECOBEE_MODEL_TO_NAME, MANUFACTURER
 
-SENSOR_TYPES: tuple[SensorEntityDescription, ...] = (
-    SensorEntityDescription(
+
+@dataclass
+class EcobeeSensorEntityDescriptionMixin:
```

Comment:
```suggestion
class EcobeeSensorEntityDescriptionMixin:
    """Represent the required ecobee entity description attributes."""
```

Reply:
I just got finished doing isort and the docstrings. Guess you beat me to it :)